### DISCUSSION TITLE
What's difference vs async-nats vs nats crate

### DISCUSSION THREAD

**Discussion by baoyachi** [nats-io/nats.rs#442] [Q&A] [answered]
* https://crates.io/crates/async-nats
* https://crates.io/crates/nats
Now, I want use nats crate with support Rust async. How do I choose？

**Comment by Jarema** [comment, marked as answer]
hey @baoyachi !

`nats` crate is sync in nature. It only has a convenience async wrapper,  but just for core ANTS.
`async-nats` is a new client, written natively async (using `tokio`, but works well with `async-std` via `compat` feature).

It is still beta, as it's pretty new, but it's already used by others.
It is very actively developed and will get JetStream support in the next releases.

I strongly suggest using `async-nats`.
